stl/src/atomic_wait.cpp

```diff
@@ -356,25 +223,10 @@ unsigned long __stdcall __std_atomic_wait_get_remaining_timeout(unsigned long lo
     return static_cast<unsigned long>(_Remaining);
 }
 
-__std_atomic_api_level __stdcall __std_atomic_set_api_level(__std_atomic_api_level _Requested_api_level) noexcept {
-#if _ATOMIC_WAIT_ON_ADDRESS_STATICALLY_AVAILABLE
-    (void) _Requested_api_level;
+// TRANSITION, ABI: preserved for binary compatibility

```

Comment:
What binary compatibility? This is just a hole punched for tests to test older APIs, and it is not called by the STL

Reply:
Correct, but we have (currently MSVC-internal) validation to ensure that the DLL's export surface doesn't change, so even useless test-only APIs like this need to be preserved. Same as this thing:

https://github.com/microsoft/STL/blob/c19507f80452efe0138fbf906e9270f8c081a2be/stl/src/mutex.cpp#L30-L32